tests/src/OneLogin/saml2_tests/response_test.py

```diff
@@ -549,11 +519,9 @@ def testValidateID(self):
         settings = OneLogin_Saml2_Settings(self.loadSettingsJSON())
         xml = self.file_contents(join(self.data_path, 'responses', 'invalids', 'no_id.xml.base64'))
         response = OneLogin_Saml2_Response(settings, xml)
-        try:
-            valid = response.is_valid(self.get_request_data())
-            self.assertFalse(valid)
-        except Exception as e:
-            self.assertEqual('Missing ID attribute on SAML Response', str(e))
```

Comment:
Now that we are able to raise exceptions, why no use them calling
 valid = response.is_valid(self.get_request_data(), None, True)